lib/jsdom/living/register-elements.js

```diff
@@ -283,6 +283,38 @@ const mappings = {
   HTMLVideoElement: {
     file: require("./generated/HTMLVideoElement.js"),
     tags: ["video"]
+  },
+  SVGCircleElement: {
+    file: require("./generated/SVGCircleElement.js"),
+    tags: ["circle"]
```

Comment:
Ok all changes done, except those test, indeed `<circle>` is invalid outside svg context. What will be the best way to implement and test that?

Once it's done, I could add the [get/set]PathData polyfill possibly, once this one have tests too